File: C_lang_Foundation_of_Programming/31_small_medium_large.c
//write a program which accpet number from user and if numer is less than 50 then print small,if it is greater than 50 and less than 100
//then print medium,if it is greater than 100 then print large.
//INPUT: 75
//OUTPUT:Medium

#include<stdio.h>
int main()
{
	int iNo=0;

	printf("Small < 50 < Medium < 100 < large \n");

	printf("Enter Number:");
	scanf("%d",&iNo);

	if(iNo<50)
		printf("small");
	else if(iNo<100)
		printf("medium");
	else
		printf("large");

	return 0;
}

File: Python_Foundation_of_Programming/31_small_medium_large.py
#write a program which accept number from user and if number is less than 50 then print small,
#if it is greater than 50 and less than 100 then medium ,if it is greater than 100 then print large.
#input: 75
#output: medium

def small_medium_large(ino):
    if ino < 50:
        print("small")
    elif ino< 100:
        print("medium")
    else:
        print("large")

ino =int(input("Enter number:"))
small_medium_large(ino)
